http_service/bugbug_http/models.py

```diff
@@ -145,3 +143,22 @@ def classify_bug(
         redis.set(change_key, bugs[bug_id]["last_change_time"])
 
     return "OK"
+
+
+def schedule_tests(revset, expiration=DEFAULT_EXPIRATION_TTL):
+    from bugbug_http.app import get_result_key
+
+    data = {
+        "tasks": ["test-macosx1014-64/debug-gtest-1proc"],
+        "groups": ["caps/test/unit/xpcshell.ini", "dom/indexedDB/test/mochitest.ini",],
+    }
```

Comment:
Not sure yet how we will handle configuring different shadow schedulers with different parameters.
I see two options:
- the configuration (confidence level) is passed to the HTTP service, and the HTTP service only returns tasks/groups with confidence higher than the one given as input;
- the HTTP service returns all tasks/groups with their associated confidence level, and the client (the shadow scheduler) does the filtering on the confidence level.

If we do the second, we're going to have to change a bit the output here (something like `["test-macosx1014-64/debug-gtest-1proc", 0.67]`).

Reply:
I think I like the second approach better. Though it will be easy to adjust on the client side so I think it's fine to keep it simple for now and adjust as needed when we run into it.